src/lib/components/collapsiblepanel/CollapsiblePanel.component.js

```diff
@@ -0,0 +1,88 @@
+//@flow
+import React from "react";
+import styled from "styled-components";
+import Button from "../button/Button.component";
+import * as defaultTheme from "../../style/theme";
+
+
+type Props = {
+  expand?: Boolean,
+  onHeaderClick?: any => void,
+  children?: any,
+  node: Array,
+};
+
+const ARROW = {
+  colapsed: 'fas fa-angle-down',
+  expanded: 'fas fa-angle-up',
+}
+
+const HeaderText = styled.div`
+  display: flex;
+`;
+
+const HeaderItem = styled.div`
+  margin-right: 20px; 
+`;
+const HeaderIcon = styled.div`
+  margin: 0 20px;
+`;
+
+const CollapsiblePanelContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px 8px 4px 4px;
+  background-color: ${defaultTheme.brand.primary};
```

Comment:
`background-color: ${props =>
    mergeTheme(props.theme, defaultTheme). backgroundContrast1};` to be themable